docs/logs/customizing-the-sdk/Program.cs

```diff
@@ -5,37 +5,53 @@
 using OpenTelemetry.Logs;
 using OpenTelemetry.Resources;
 
-namespace CustomizingTheSdk;
-
-public class Program
+var loggerFactory = LoggerFactory.Create(builder =>
 {
-    public static void Main()
+    builder.AddOpenTelemetry(logging =>
     {
-        using var loggerFactory = LoggerFactory.Create(builder =>
-        {
-            builder.AddOpenTelemetry(options =>
-            {
-                options.IncludeScopes = true;
-                options.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(
-                    serviceName: "MyService",
-                    serviceVersion: "1.0.0"));
-                options.AddConsoleExporter();
-            });
-        });
-
-        var logger = loggerFactory.CreateLogger<Program>();
-
-        logger.LogInformation("Hello from {name} {price}.", "tomato", 2.99);
-        logger.LogWarning("Hello from {name} {price}.", "tomato", 2.99);
-        logger.LogError("Hello from {name} {price}.", "tomato", 2.99);
-
-        // log with scopes
-        using (logger.BeginScope(new List<KeyValuePair<string, object>>
-        {
-            new KeyValuePair<string, object>("store", "Seattle"),
-        }))
-        {
-            logger.LogInformation("Hello from {food} {price}.", "tomato", 2.99);
-        }
-    }
+        logging.IncludeScopes = true;
+        logging.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(
+            serviceName: "MyService",
+            serviceVersion: "1.0.0"));
+        logging.AddConsoleExporter();
+    });
+    builder.AddFilter<OpenTelemetryLoggerProvider>("*", LogLevel.Information);
```

Comment:
a code comment to explain what this line does?

Reply:
I've scoped down this PR as I realized there are more things to be explained, will try to address this in a separate PR.